Makefile

```diff
@@ -13,6 +13,8 @@ STACK_PROTECTOR        ?= on
 BUILDER_TAG             = $(ROS_DISTRO)$(shell \
   if [ $(ROS_DISTRO) = "noetic" ] && [ $(ALPINE_VERSION) != "3.11" ]; then \
     echo -n ".v$(ALPINE_VERSION)"; \
+	elif [ $(ROS_DISTRO) = "humble" ]; then \
+		echo -n ".v$(ALPINE_VERSION)"; \
   fi)
```

Comment:
Noetic 3.11 is already dropped by https://github.com/seqsense/aports-ros-experimental/pull/819
so it can be always `$(ROS_DISTRO).v$(ALPINE_VERSION)`